### PR TITLE
Mobile portfolio Works and About sections

### PR DESCRIPTION
**Pull request by Nyame-Wolf**
# Mobile portfolio Works and About sections
- Added works sections. Used Css grid to place the cards and flex box to place card contents
-  styled css list to buttons
- enabled hover effect on list buttons.
# Reviewers Recommendations:
- Added semantic tag article to every card in works section
- Added first card to grid to make all works section cards grid items
- Changed first section's paragraph color to display it
- fixed first card's see project button to appear on the left
- Fixed see project button for the second to 6th card to appear on the left and fit container width
- fixed project description on Readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,8 +8,9 @@
```
Comment (Ismailco) [inline README.md:8]: - [ ] Kindly add a project description to the README, so your documentation is more professional

File: index.html [hunk 2]
```diff
@@ -63,6 +67,184 @@ <h1 class="item">Hey There. I’m Mumenya</h1>
           </ul>
         </div>
       </section>
+      <section>
+        <div id="item1">
```
Comment (Ismailco) [inline index.html:71]: - [ ] For better practices you can use semantic stages here e.g. ```<article>```, please try to minimalize the use of ```<div>``` in your code instead use ```<section>``` and ```<article>```

File: index.html [hunk 3]
```diff
@@ -63,6 +67,184 @@ <h1 class="item">Hey There. I’m Mumenya</h1>
           </ul>
         </div>
       </section>
+      <section>
```
Comment (Ismailco) [inline index.html:70]: - [ ] You can use ```grid``` to style all the projects, starting with this project

File: index.html [hunk 4]
```diff
@@ -63,6 +67,184 @@ <h1 class="item">Hey There. I’m Mumenya</h1>
...
+          </ul>
+        </div>
+        <div class="works grid">
+          <div class="item2">
+            <ul class="button-orange link">
+              <li><a href="#">See Project</a></li>
+            </ul>
+
+            <div class="works-description">
+              <h4>Profesional Art Printing Data</h4>
```
Comment (Ismailco) [inline index.html:98]: - [ ] [OPTIONAL] Please consider using the same ```hx``` for the title of the projects

File: style.css [hunk 5]
```diff
@@ -128,3 +131,263 @@ h1 {
   background-position: bottom right;
   background-repeat: no-repeat;
 }
+
+.grid {
```
Comment (Ismailco) [inline style.css:135]: - [ ] You need to add the paragraph for the first project and align the ```See Project``` button to the left, please stick to the Figma design as much as possible

|Your design|Figma design|
|--|--|
|Screenshot_20220527_192857|Screenshot_20220527_192956|

File: style.css [hunk 6]
```diff
@@ -128,3 +131,263 @@ h1 {
...
+  display: grid;
+  grid-template-rows: auto;
+}
+
+#item1 {
```
Comment (Ismailco) [inline style.css:140]: - [ ] The ```See project``` button should be at the bottom of the project card, please use the propper ``` margin/spacing``` and modify your design so it is as close as much to the Figma design

|Your design|Figma design|
|--|--|
|Screenshot_20220527_192836|Screenshot_20220527_192940|

File: style.css [hunk 7]
```diff
@@ -128,3 +131,263 @@ h1 {
...
+.about-title {
+  font-family: "Inter", sans-serif;
+  font-style: bold;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+}
+
+.about-li {
```
Comment (Ismailco) [inline style.css:363]: - [ ] The ```languages```, ```skills```, and ```framework``` items should be centered, please implement this and stick to the design as much as possible

|Your design|figma design|
|-|-|
|Screenshot_20220527_192818|Screenshot_20220527_192921|

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @Nyame-Wolf,

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 8]
```diff
@@ -63,6 +67,171 @@ <h1 class="item">Hey There. I’m Mumenya</h1>
...
+              <li><a href="#">bootstrap</a></li>
+
+              <li><a href="#">Ruby</a></li>
+            </ul>
+            <button type="submit" class="button-orange small-mg">
+              See Project
+            </button>
+          </div>
+        </article>
+        <!-- <div class="grid"> -->
```
Comment (Div685) [inline index.html:94]: - [ ] [Optional] Please remove commented code. It makes your repo and your project unprofessional. if you still want to check the previous changes you made you can go back to the previous commits and check.

**Review by Div685 — APPROVED** [review]
Hi, **@Nyame-Wolf**, I'm Divyesh your reviewer for this PR :man_technologist: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Keep on coding! :muscle:

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Code review by **Divyesh Patel**.

_(Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.)_

------
>_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**